engines/gob/detection/tables_adi2.h

```diff
@@ -25,7 +25,7 @@
  *
  */
 
-/* Detection tables for the ADI / Addy 2 series. */
+/* Detection tables for the ADI 2 series. */
```

Comment:
This commit was also in the other PR. Not that it matters much, but I didn't add it on purpose because the game is known both as ADI and Addy (e.g. the german version says "Addy" in the main menu)

Reply:
Its nearly correct, Addy was first used for the german release of ADI 4, ADI 2 in German is known as ADI Spielerisch lernen / ADI 2

Reply:
Ah, these are the Adi entries, not Adibou. Ok, sorry for the noise